.env.example

```
@@ -2,3 +2,4 @@ REACT_APP_API_DOMAIN=http://localhost:8000
 REACT_APP_ENV='dev'
 REACT_APP_DEVELOP_BUILD_VERSION=
 REACT_APP_LAUNCHDARKLY_CLIENT_ID=65df9142fbf14e10954ac55d
+REACT_APP_WEB_URI=http://localhost:3000
```

3000 is the admin app port; shouldn't it be this?
```suggestion
REACT_APP_WEB_URI=http://localhost:5173
```

---

Good catch 🙏 I made the same mistake in my `.env` file, but forgot to fix it here